src/server/managers/AppListenerManager.ts

```diff
@@ -394,28 +410,30 @@ export class AppListenerManager {
         return prevented;
     }
 
-    private async executePostMessageDelete(data: IMessage): Promise<void> {
-        const cfMsg = new Message(Utilities.deepCloneAndFreeze(data), this.manager);
+    private async executePostMessageDelete(data: IMessageDeleteContext): Promise<void> {
+        const context = Utilities.deepCloneAndFreeze(data);
+        const {message} = context;
 
         for (const appId of this.listeners.get(AppInterface.IPostMessageDeleted)) {
             const app = this.manager.getOneById(appId);
 
             let continueOn = true;
-            if (app.hasMethod(AppMethod.CHECKPOSTMESSAGEDELETED)) {
-                continueOn = await app.call(AppMethod.CHECKPOSTMESSAGEDELETED,
-                    cfMsg,
+            if (app.hasMethod(AppMethod.CHECK_POST_MESSAGE_DELETED)) {
+                continueOn = await app.call(AppMethod.CHECK_POST_MESSAGE_DELETED,
+                    message,
```

Comment:
The definition above said this is context but you're still passing message. Continue passing the message here but you can add the context to the method's definition.